torch/csrc/distributed/rpc/functions.cpp

```diff
@@ -50,6 +59,47 @@ Message processRequestBlocking(Message&& request) {
       }
       break;
     }
+    case MessageType::REMOTE_CALL: {
+      ScriptRemoteCall src = ScriptRemoteCall::fromMessage(request);
+
+      auto rrefId = RRefId::fromIValue(src.retRRefId());
+      auto forkId = ForkId::fromIValue(src.retForkId());
+      TORCH_CHECK(rrefId != forkId, "Does not support remote call to self.");
+
+      auto& ctx = RRefContext::getInstance();
+      auto ownerRRef = ctx->getOrCreateOwnerRRef<IValue>(rrefId);
+
+      // TODO: make this asynchronous
+      auto stack = src.stack();
```

Comment:
Does this need to be a copy? Looks like `src` has a short life time, so could be mutated in place with `auto&`.

Reply:
added a `stackRef` API to make it explicit.